tutorminio/templates/minio/tasks/minio/init.sh

```diff
@@ -1,9 +1,36 @@
+# Verify that `MINIO_GCS_APPLICATION_CREDENTIALS` and `MINIO_GCS_APPLICATION_ID` are defined if `MINIO_GATEWAY == "gcs"`
+{% if MINIO_GATEWAY == "gcs" %}
+  {% if not MINIO_GCS_APPLICATION_CREDENTIALS %}
+    echo "⚠️  MINIO_GCS_APPLICATION_CREDENTIALS is not defined. It is required to when MINIO_GATEWAY is set to gcs."
+    exit 1
+  {% endif %}
+  {% if not MINIO_GCS_APPLICATION_ID %}
+    echo "⚠️  MINIO_GCS_APPLICATION_ID is not defined. It is required to when MINIO_GATEWAY is set to gcs."
+    exit 1
+  {% endif %}
+{% endif %}
+
 mc config host add minio http://minio:9000 {{ OPENEDX_AWS_ACCESS_KEY }} {{ OPENEDX_AWS_SECRET_ACCESS_KEY }} --api s3v4
-mc mb --ignore-existing minio/{{ MINIO_BUCKET_NAME }} minio/{{ MINIO_FILE_UPLOAD_BUCKET_NAME }} minio/{{ MINIO_VIDEO_UPLOAD_BUCKET_NAME }}
 
-{% if MINIO_GATEWAY != "azure" %}
+BUCKETS=(
+    "{{ MINIO_BUCKET_NAME }}"
+    "{{ MINIO_FILE_UPLOAD_BUCKET_NAME }}"
+    "{{ MINIO_VIDEO_UPLOAD_BUCKET_NAME }}"
+)
+
```

Comment:
We need to create 3 buckets in advance

Reply:
In fact, it is interesting the Video upload bucket is not listed among the bucket in the Readme file:
MINIO_BUCKET_NAME (default: "openedx")
MINIO_FILE_UPLOAD_BUCKET_NAME (default: "openedxuploads")
-----
MINIO_VIDEO_UPLOAD_BUCKET_NAME ?

Reply:
Ok I'll create the buckets in preparation for the tests.

Reply:
To keep it consistent in the way we are naming our infrastructure, I named them in the following way:

<project_id>-minio
<project_id>-minio-uploads
<project_id>-minio-video-uploads

Example:

![Screenshot from 2023-11-01 10-41-28](https://github.com/correlation-one/tutor-minio/assets/3267469/c5e5f92d-f766-4164-b2de-d526d5b9fbe9)

I believe it's clear what the are about and unique (project id guaranties that).

Reply:
done for staging and sandbox

Reply:
It seems there is no need for creating the buckets in advance. If they don't exist, they `mc` creates them.